packages/beeq/src/components/card/scss/bq-card.variables.scss

```diff
@@ -0,0 +1,14 @@
+/* -------------------------------------------------------------------------- */
+/*                   Card custom properties                  */
+/* -------------------------------------------------------------------------- */
+
+:host {
+  /**
+   * @prop --bq-card--padding - Card padding
+   * @prop --bq-card--background - Card background
+   * @prop --bq-card--border-radius - Card border radius
+   */
+  --bq-card--padding: theme('spacing.l');
+  --bq-card--background: theme('colors.bg.secondary');
+  --bq-card--border-radius: theme('borderRadius.m');
```

Comment:
I would recommend adding CSS variables for border-style, border-width, and border-color.